{"issue_type": "request_error", "error_msg": "HTTPConnectionPool(host='vergennes.org', port=80): Max retries exceeded with url: /robots.txt (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fc40dafb460>, 'Connection to vergennes.org timed out. (connect timeout=15)'))"}